### DISCUSSION TITLE
How to connect to another database on the same project using the database Id

### DISCUSSION THREAD

**Discussion by qdubois** [firebase/firebase-js-sdk#7616] [Q&A]
Hello,

I'm looking how to connect to another database on the same project using the databaseId with the firebase sdk?

I haven't found anything in the documentation.

thanks a lot

Quentin

**Comment by ryanmio** [comment]
Use initializeApp for multiple Firebase databases.

1. Initialize First Database
```javascript
const primaryApp = firebase.initializeApp(primaryConfig);
const primaryDatabase = firebase.database(primaryApp);
```

2: Initialize Second Database with databaseId
```javascript
const secondaryConfig = { ...primaryConfig, databaseId: "your-database-id" };
const secondaryApp = firebase.initializeApp(secondaryConfig, "secondary");
const secondaryDatabase = firebase.database(secondaryApp);
```
